components/locale_canonicalizer/benches/locale_canonicalizer.rs

```diff
@@ -0,0 +1,52 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+use criterion::{criterion_group, criterion_main, Criterion};
+use icu_locale_canonicalizer::LocaleCanonicalizer;
+use icu_locid::Locale;
+
+fn maximize_bench(c: &mut Criterion) {
+    let provider = icu_testdata::get_provider();
+    let lc = LocaleCanonicalizer::new(&provider).unwrap();
+
+    let mut group = c.benchmark_group("likelysubtags");
+
+    let locales: Vec<Locale> = vec![
+        "en-US",
+        "en-GB",
+        "es-AR",
+        "it",
+        "zh-Hans-CN",
+        "de-AT",
+        "pl",
+        "fr-FR",
+        "de-AT",
+        "sr-Cyrl-SR",
+        "nb-NO",
+        "fr-FR",
+        "mk",
+        "uk",
+        "und-PL",
+        "und-Latn-AM",
+        "ug-Cyrl",
+        "sr-ME",
+        "mn-Mong",
+        "lif-Limb",
+    ]
+    .iter()
+    .map(|s| s.parse().unwrap())
+    .collect();
```

Comment:
could you move that data to json fixture? You can find examples of similar ones here: https://github.com/unicode-org/icu4x/tree/master/components/locid/benches/fixtures

The reason is that we'd like to make it easy to test our tests/benchmarks against ICU4C, or FFI outputs of our crates. Having this hardcoded in Rust limits our ability to do so.

Reply:
https://github.com/unicode-org/icu4x/issues/410

Reply:
I'll move this to a fixture for now, pending resolution of #410.